start_admin_sshd.sh

```diff
@@ -63,6 +68,10 @@ EOF
   chmod 644 "${proxy_profile}"
 }
 
+# Create local user
+echo "${LOCAL_USER} ALL=(ALL) NOPASSWD: ALL" > "/etc/sudoers.d/${LOCAL_USER}"
+chmod 440 "/etc/sudoers.d/${LOCAL_USER}"
+useradd -m -G users,api "${LOCAL_USER}"
```

Comment:
In the original Dockerfile the `useradd` came after creating the sudoers file. Does it make more sense to move it to the top here?

Reply:
I'm not sure it matters; the sudoers file isn't used until the user tries to sudo.  It makes sense to me to keep the commands together like you have...